### PR TITLE
Updates to CAF

### PR DESCRIPTION
**Pull request by gputnam**
More header information, rationalize the per-cryostat truth matching a bit, add in more information for the MeVPrtl generator.

### CODE REVIEW

File: sbnanaobj/StandardRecord/classes_def.xml [hunk 1]
```diff
@@ -62,7 +63,9 @@
    <version ClassVersion="10" checksum="1169960296"/>
   </class>
 
-  <class name="caf::SRTrackTruth" ClassVersion="11">
+  <class name="caf::SRTrackTruth" ClassVersion="13">
+   <version ClassVersion="13" checksum="2098868401"/>
+   <version ClassVersion="12" checksum="1268322903"/>
```
Comment (PetrilloAtWork) [inline sbnanaobj/StandardRecord/classes_def.xml:68]:
```suggestion
  <class name="caf::SRTrackTruth" ClassVersion="12">
   <version ClassVersion="12" checksum="2098868401"/>
```

**Review by PetrilloAtWork — CHANGES_REQUESTED** [review]
Spurious version checksum; the rest is good.

File: sbnanaobj/StandardRecord/SRHeader.cxx [hunk 2]
```diff
@@ -23,7 +23,9 @@ namespace caf
   ngenevt(0),
   pot(0.),
   mctype(caf::kMCUnknown),
-  det(std::numeric_limits<Det_t>::signaling_NaN())
+  det(std::numeric_limits<Det_t>::signaling_NaN()),
```
Comment (cjbackhouse) [inline sbnanaobj/StandardRecord/SRHeader.cxx:26]: I don't think this makes sense. I wish std::numeric_limits would refuse to compile for integer types that don't actually have a NaN. Here I would use `-1`.

Comment (gputnam) [reply sbnanaobj/StandardRecord/SRHeader.cxx:26]: This PR does not have to do with that line of code, but I can change it.

Comment (cjbackhouse) [reply sbnanaobj/StandardRecord/SRHeader.cxx:26]: I guess you added a comma to it :smile:

**Review by FernandaPsihas — APPROVED** [review]

**Review by PetrilloAtWork — APPROVED** [review]

**Pull request merged**